### PR TITLE
Update citizens-house.md

### PR DESCRIPTION
**Pull request by JSeiferth**
Updating Citizens House overview doc to reflect latest thinking and language

### CODE REVIEW

File: src/docs/governance/citizens-house.md [hunk 1]
```diff
@@ -6,56 +6,49 @@ lang: en-US
...
 
 ## A Commitment to Experimentation
+<p align="center">
+   <img width="70%" src="https://github.com/ethereum-optimism/community-hub/assets/43515441/4a4ea307-dbe9-4034-89fb-53d320ad740c">
+</p>
 
 RetroPGF is a longterm bet to build the future Optimists want to see. 
 The Collective will conduct regular rounds of RetroPGF, each different from the last. 
 This is an emergent process that will take community participation to grow and prune. 
+1. [RetroPGF round 1](./retropgf-1.md) took place at the end of 2021 and distribute $1 million across 58 projects.
```
Comment (bdresser) [inline src/docs/governance/citizens-house.md:47]: distributed*

Comment (bdresser) [reply src/docs/governance/citizens-house.md:47]: same below

File: src/docs/governance/citizens-house.md [hunk 2]
```diff
@@ -6,56 +6,49 @@ lang: en-US
...
+<p align="center">
+   <img width="70%" src="https://github.com/ethereum-optimism/community-hub/assets/43515441/4a4ea307-dbe9-4034-89fb-53d320ad740c">
+</p>
 
 RetroPGF is a longterm bet to build the future Optimists want to see. 
 The Collective will conduct regular rounds of RetroPGF, each different from the last. 
 This is an emergent process that will take community participation to grow and prune. 
+1. [RetroPGF round 1](./retropgf-1.md) took place at the end of 2021 and distribute $1 million across 58 projects. 
+2. [RetroPGF round 2](./retropgf-2.md) took place in Q1 2023 and distribute 10m OP token across 195 projects.
+3. [RetroPGF round 3](./retropgf-3.md) will take place in Q3/Q4 2023 and will distribute 30m OP to contributions that have supported the development and adoption of Optimism.
```
Comment (bdresser) [inline src/docs/governance/citizens-house.md:49]: Q4 i think is cleaner - will take place after Q3

File: src/docs/governance/citizens-house.md [hunk 3]
```diff
@@ -6,56 +6,49 @@ lang: en-US
...
 Funding for RetroPGF can come from:
+- RetroPGF token allocation [(20% of the initial OP supply)](./allocations.md#retroactive-public-goods-funding)
+- Allocation of surplus protocol revenue 
 
-- **[20% of the initial OP supply](./allocations.md#retroactive-public-goods-funding)**
-- Redeployment of some profit generated by the Optimism protocol
-
-Many values-aligned projects on Optimism have also chosen to contribute a portion of their revenue to retroPGF. 
+In future stages, the Citizens’ House role will expand beyond RetroPGF. 
+For example, in addition to voting on RetroPGF, the Citizens’ House will work alongside the Token House to govern allocation of surplus protocol revenue, collaborate on criteria for participation in the Citizens’ House, and engage in a system of checks and balances to enforce the Collective’s Codes of Conduct.
```
Comment (bdresser) [inline src/docs/governance/citizens-house.md:17]: link to "future of gov" blog post here?

File: src/docs/governance/citizens-house.md (hunk 2 above)
Comment (cristinaja) [reply src/docs/governance/citizens-house.md:49]: please change "distributed"/"distribute" to "granted"/"grant" in round 2 and 3, and anywhere else in connection with the token

**Review by sbvegan — APPROVED** [review]

**Pull request merged**